Subject: Fix use of PAGE_SIZE
From: Marc Kleine-Budde <mkl@pengutronix.de>

Don't rely on kernel-header, Use sysconf to get PAGE_SIZE.

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>

---
 lib/direct/system.c |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

Index: DirectFB-1.0.1/lib/direct/system.c
===================================================================
--- DirectFB-1.0.1.orig/lib/direct/system.c
+++ DirectFB-1.0.1/lib/direct/system.c
@@ -36,7 +36,9 @@
 
 #if HAVE_ASM_PAGE_H
 #include <asm/page.h>
-#else
+#endif
+
+#ifndef PAGE_SIZE
 #define PAGE_SIZE   sysconf( _SC_PAGESIZE )
 #endif
 
